dotCMS/src/main/java/com/dotcms/security/ContentSecurityPolicyUtil.java

```diff
@@ -0,0 +1,192 @@
+package com.dotcms.security;
+
+import static com.dotcms.util.CollectionsUtils.map;
+
+import com.dotcms.api.web.HttpServletRequestThreadLocal;
+import com.dotcms.repackage.org.apache.axiom.om.util.Base64;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.PageMode;
+import com.dotmarketing.util.UtilMethods;
+import java.util.Map;
+import java.util.Map.Entry;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.commons.lang.RandomStringUtils;
+
+public class ContentSecurityPolicyUtil {
+
+    final static Map<String, ContentSecurityPolicyResolver> contentSecurityPolicyResolvers;
+    public static final int RANDOM_STRING_LENGTH = 20;
+
+    private static String NONCE_REQUEST_ATTRIBUTE = "NONCE_REQUEST_ATTRIBUTE";
```

Comment:
it does not looks such as constant

Reply:
done https://github.com/dotCMS/core/pull/21337/commits/b1ced9fd711e813176c2dad89b70d29818946963